src/rt/CMakeLists.txt

```diff
@@ -332,4 +332,22 @@ if(NOT DEFINED VERONA_RT_ONLY_HEADER_LIBRARY)
       add_test(${TESTNAME} func-sys-backpressure --cores ${CORES} --seed ${SEEDLOWER} --seed_upper ${SEEDUPPER})
     endforeach()
   endforeach()
+
+  foreach(CORES 1 2 3 4)
+    foreach(SEED RANGE 1 2)
+      MATH(EXPR SEEDLOWER "${SEED} * 10")
+      MATH(EXPR SEEDUPPER "((${SEED} + 1) * 10) - 1")
+      SET (TESTNAME "runtime/func-sys-runtimepause_${CORES}_${SEEDLOWER}")
+      add_test(${TESTNAME} func-sys-runtimepause --cores ${CORES} --seed ${SEEDLOWER} --seed_upper ${SEEDUPPER}  --log-all)
+    endforeach()
+  endforeach()
+
+  foreach(CORES 1 2 3 4)
+    foreach(SEED RANGE 1 2)
+      MATH(EXPR SEEDLOWER "${SEED} * 10")
+      MATH(EXPR SEEDUPPER "(${SEED} * 10) + 9")
+      SET (TESTNAME "runtime/func-con-runtimepause_${CORES}_${SEEDLOWER}")
+      add_test(${TESTNAME} func-con-runtimepause --cores ${CORES} --seed ${SEEDLOWER} --seed_upper ${SEEDUPPER})
```

Comment:
Should this be missing the `--log-all` option when the other added tests include it?

Reply:
The `--log-all` was for debugging, and shouldn't have been committed.